docker/lisk_service/docker-compose.core.yml

```diff
@@ -14,6 +14,8 @@ services:
     env_file:
       - ./lisk_service/env/common.env
       - ./lisk_service/env/core.env
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
Where does `host-gateway` come from?

Reply:
It's more magic :grin:

Reply:
I think it's pretty new too. Might be the very latest version of Docker even

Reply:
And this one only works on linux? Or why do we not use it directly in `LISK_CORE_*`?

Reply:
This looks like the original PR https://github.com/moby/moby/pull/40007 
It's weird, but it looks like `host-gateway` can't just be used directly